pyanaconda/threading.py

```diff
@@ -168,7 +168,7 @@ def raise_if_error(self, name):
         with self._errors_lock:
             exc_info = self._errors.pop(name)
         if exc_info:
-            raise exc_info[0](exc_info[1]).with_traceback(exc_info[2])
+            raise exc_info[1].with_traceback(exc_info[2])
```

Comment:
Do we need the `.with_traceback()` part, isn't that already part of the instance?

Reply:
Ha, nice idea. It didn't occur to me that might be the case. I'll check that!

Reply:
Yes, that works too! Less code \o/